integration_tests/suite/test_call_log_generation.py

```diff
@@ -1,5 +1,6 @@
 # Copyright 2022-2023 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
+# flake8: noqa: W291
```

Comment:
Because flake8 was complaining about trailing spaces in the test data strings. I admit ignoring all trailing whitespaces is maybe not ideal, and I would expect string contents to be excluded.

Reply:
@DrPyser I don't understand why removing them is a problem
see: https://github.com/wazo-platform/wazo-call-logd/pull/174/commits/aee29f8829ede36cb1d91eeb0fc454ce85e77514

Reply:
I just considered it worthless busywork. flake8 has no business checking the content of data strings. The point of being able to simply paste sql table is to make it easy to create and maintain tests by minimizing manual data-entering. Why spend time formatting data beyond basic readability?